module/web/media/js/pyplex/recaptcha-v2-interactive.js

```diff
@@ -0,0 +1,90 @@
+﻿/*
```

Comment:
This file contains tabs instead of spaces and is identical to module/web/media/js/classic/recaptcha-v2-interactive.js
Can you put it in the parent directory and reuse it in both themes?